Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCPeerConnectionBackend.h

```diff
@@ -95,7 +95,7 @@ class LibWebRTCPeerConnectionBackend final : public PeerConnectionBackend {
     ExceptionOr<Ref<RTCRtpSender>> addTrack(MediaStreamTrack&, FixedVector<String>&&) final;
     void removeTrack(RTCRtpSender&) final;
 
-    ExceptionOr<Ref<RTCRtpTransceiver>> addTransceiver(const String&, const RTCRtpTransceiverInit&) final;
+    ExceptionOr<Ref<RTCRtpTransceiver>> addTransceiver(const String&, const RTCRtpTransceiverInit&, IgnoreNegotiationNeededFlag = IgnoreNegotiationNeededFlag::No) final;
```

Comment:
Do we need the ` = IgnoreNegotiationNeededFlag::No`?

Reply:
It can be made explicit, no problem :)